CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
-# 5.0.0 (Unreleased)
+# 4.5.0 (Unreleased)
 #### Notes
 Added the capability to set a connection timeout when connecting to the HPE OneView Appliance
-Major release which extends support of the SDK to OneView Rest API version 600 (OneView v4.0).
+Extends support of the SDK to OneView Rest API version 600 (OneView v4.0).
 
 #### Features supported with current release:
-- Connection Templates
-- FC Networks
-- Logical Interconnect Groups
-- Interconnect Types
-- SAS Logical Interconnect Groups
+- Connection template
+- FC network
+- Logical interconnect group
+- Interconnect type
```

Comment:
`Interconnect types` should likely come before `Logical Interconnect` so that the list is ordered alphabetically